packages/manager/modules/pci/src/components/users/download-rclone/download-rclone.controller.js

```diff
@@ -2,48 +2,44 @@ import first from 'lodash/first';
 import get from 'lodash/get';
 import map from 'lodash/map';
 
-import { DOWNLOAD_FILENAME, DOWNLOAD_TYPE } from './download-rclone.constants';
+import {
+  DOWNLOAD_FILENAME,
+  DOWNLOAD_TYPE,
+  DOWNLOAD_RCLONE_FILETYPE,
+  DOWNLOAD_RCLONE_FILENAME,
+  STATE_TYPE,
+} from './download-rclone.constants';
 
 const { saveAs } = require('file-saver');
 
 export default class PciUsersDownloadRcloneController {
   /* @ngInject */
-  constructor(
-    $compile,
-    $translate,
-    ovhManagerRegionService,
-    PciProjectsProjectUsersService,
-  ) {
+  constructor($compile, $translate, ovhManagerRegionService) {
     this.$compile = $compile;
     this.$translate = $translate;
     this.ovhManagerRegionService = ovhManagerRegionService;
-    this.PciProjectsProjectUsersService = PciProjectsProjectUsersService;
   }
 
   $onInit() {
     this.isLoading = false;
-
     this.regions = map(this.regions, (region) => ({
       id: region,
       label: this.ovhManagerRegionService.getTranslatedMicroRegion(region),
     }));
     this.region = first(this.regions);
-    this.hasGlobalRegions = this.PciProjectsProjectUsersService.checkGlobalRegion(
-      this.regions,
-    );
+    this.hasGlobalRegions = this.checkGlobalRegionCallBack(this.regions);
   }
 
   downloadRclone() {
     this.isLoading = true;
-    return this.PciProjectsProjectUsersService.downloadRclone(
-      this.projectId,
-      this.user,
-      this.region.id,
-    )
+    this.fileType =
+      this.state === STATE_TYPE ? DOWNLOAD_RCLONE_FILETYPE : DOWNLOAD_TYPE;
+    this.fileName =
+      this.state === STATE_TYPE ? DOWNLOAD_RCLONE_FILENAME : DOWNLOAD_FILENAME;
+    return this.downloadCallBack(this.region.id)
```

Comment:
**nitpick(non-blocking):** `download-rclone` component can accept `fileType` and `fileName` as inputs instead of deciding it in the component based on the place where it is used.

Reply:
Okay Anoop will make the changes